docs/reference/cli-reference.md

```diff
@@ -241,6 +241,7 @@ $ infra grants add johndoe@example.com infra --role admin
 ### Options
 
 ```
+  -f, --force         Force add even if requested resources do not exist
```

Comment:
I agree. The flag is also shared between missing users/groups and missing destinations. Ideally there should be multiple flags in case user wants to allow a missing user but not a missing destination, or vice versa.

Reply:
I'm not sure about `--allow-missing`. It seems like that would be really easy to miss when combing through documentation, and also feels slightly more ambiguous. I can think of a half dozen commands that use some variation of `-f` or `--force`, such as:

```
brew install -f
cp -f
dpkg -i --force-all
ln -f
rm -f
rpm install --force
```

Reply:
All of those examples have to do with files, right? I think in that context it might be a bit more obvious. 

In many of those cases force means "allow the overwrite of an existing thing", so a user could pretty easily interpret force to mean "overwrite the existing grant".  We may even want to use it for that purpose in the future.

I think something that is more descriptive about which condition we are permitting would be a bit more obvious. Maybe `--allow` is the wrong term. `--permit-unknown-resource` ?

Reply:
I think `--force` works as well. It's good we considered some alternatives, but if they don't feel like a good fit, `--force` should be obvious enough with the description we have here.  I think we should avoid `-f` for now, as we do use that to mean `--config-file` on other commands.